rust/main/agents/relayer/src/server/message_retry.rs

```diff
@@ -58,12 +120,22 @@ mod tests {
         let addr = server.local_addr();
         tokio::spawn(server);
 
-        (addr, broadcast_tx.subscribe())
+        let retry_req_rx = broadcast_tx.subscribe();
+
+        TestServerSetup {
+            socket_address: addr,
+            retry_req_rx,
+        }
     }
 
+    #[tracing_test::traced_test]
```

Comment:
this is so useful, always wondered why we couldn't see the logs 🔥 great addition